modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -439,3 +436,23 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'modular_citadel/icons/obj/clothing/cit_hats.dmi'
+	desc = "A red fez like hat, has a yellow lining on the side."
+	item_state = "telegramhat"
+	icon_state = "telegramhat"
+	armor = list("melee" = 0, "bullet" = -500, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //Dont shoot the messager!
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/telegramhat
+	name = "Singing Telegram Suit"
+	icon = 'modular_citadel/icons/obj/clothing/cit_clothes.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/obj/clothing/turtlenecks.dmi'
+	desc = "'Hello, I am, your singing, telegram!' \
```

Comment:
```suggestion
	desc = "'Hello, I am your singing telegram!' \
```
Unless it's about phonetic pauses.

Reply:
In the movie is phonetic pauses so ya <3